docs/contributing.md

```diff
@@ -28,6 +28,43 @@ If you want to contribute and don't know where to start, here is a step-by-step
 6. Run `yarn` - install dependencies
 7. Run `yarn dev` - URL will be logged
 
+## Login locally
+
+When downloading the app you will have two options for login and they both work in different ways.
+
+### Dev login
+
+Dev login will create you a dev account so you don't need to be online or create a github app.
+To enable you must have two lines in your `.env`:
```

Comment:
or environment (.env file is not the only way, so that may be a tad confusing)

Reply:
I'm gonna leave it this way since I would have find it more confusing if it didn't say the name of the file